jpa/src/main/java/org/jboss/as/jpa/management/DynamicManagementStatisticsResource.java

```diff
@@ -211,7 +211,7 @@ public Resource navigate(PathAddress address) {
         }
         catch( IllegalStateException e) {  // WFLY-2436 ignore unexpected exceptions (e.g. JIPI-27 may throw an IllegalStateException)
             JPA_LOGGER.unexpectedStatisticsProblem(e);
-            return Collections.EMPTY_SET;
+            return Collections.emptySet();
```

Comment:
Type safety: The expression of type Set needs unchecked conversion to conform to Set<String>

`EMPTY_SET` is of type `Set` whereas `emptySet` is a generic method that returns `Set<T>` (which happens to be the same object).